pkg/backend/path_config.go

```diff
@@ -139,6 +141,11 @@ var configFields = map[string]*framework.FieldSchema{
 		Type:        framework.TypeKVPairs,
 		Description: "Specifies any provider-specific options.",
 	},
+	"refresh_interval": {
+		Type:        framework.TypeInt,
+		Description: "Specifies the interval in seconds between credential refresh, disabled if 0.",
+		Default:     60,
```

Comment:
I'd probably make this a constant and share it with the `refreshInterval := 60` code in the descriptor so any changes will be synchronized.

Reply:
Good idea, will do.